docs/advanced-usage/connecting-cloud-storage/amazon-s3.md

```diff
@@ -0,0 +1,111 @@
+Integrations can be established to Amazon S3.
+
+To get started, ensure you have admin access within Kolena.
```

Comment:
We should consider adding an FAQ explaining what an "admin" is and how you become one.